Doc/library/os.rst

```diff
@@ -706,7 +706,7 @@ process and user.
    :func:`socket.gethostname`  or even
    ``socket.gethostbyaddr(socket.gethostname())``.
 
-   .. availability:: recent flavors of Unix.
+   .. availability:: Unix.
```

Comment:
Should these get a note like `Might not be available on less recent flavors of Unix.` or `Might not be available on some flavors of Unix.`?  Even better would be to be more specific (e.g. `Not available on Unix flavors that ...`).

Reply:
`uname` is available on all Unix-like platforms that we support. It was standardized in IEEE Std 1003.1-2001, https://pubs.opengroup.org/onlinepubs/009604599/functions/uname.html . Every POSIX-like OS released in the last 20 years should have it.